packages/jazz-tools/src/tools/tests/inbox.test.ts

```diff
@@ -357,28 +364,285 @@ describe("Inbox", () => {
     const errorLogSpy = vi.spyOn(console, "error").mockImplementation(() => {});
 
     // Subscribe to inbox messages
-    const unsubscribe = receiverInbox.subscribe(
-      Message,
-      async () => {
-        spy();
-      },
-      { retries: 2 },
-    );
+    const unsubscribe = receiverInbox.subscribe(Message, async () => {
+      spy();
+    });
 
     await waitFor(() => {
       const [failed] = Object.values(receiverInbox.failed.items).flat();
 
-      expect(failed?.value.errors.length).toBe(3);
+      expect(failed?.value.errors.length).toBe(1);
     });
 
     expect(spy).not.toHaveBeenCalled();
     unsubscribe();
 
     expect(errorLogSpy).toHaveBeenCalledWith(
-      "Error processing inbox message",
-      expect.any(Error),
+      new Error("Inbox: message co_z123234 is unavailable"),
     );
 
     errorLogSpy.mockRestore();
   });
+
+  it("should skip processed messages when a large inbox is restarted", async () => {
+    cojsonInternals.setMaxRecommendedTxSize(100);
+
+    const sender = await createJazzTestAccount();
+    const receiver = await createJazzTestAccount();
+
+    await receiver.$jazz.waitForAllCoValuesSync();
```

Comment:
Is it not necessary for wait for sender as well?

Reply:
We only load data from the receiver account